### DISCUSSION TITLE
[@types/minimatch] error TS1005: ',' expected.

### DISCUSSION THREAD

**Discussion by nitin1104** [DefinitelyTyped/DefinitelyTyped#62362] [Issues with a @types package] [answered]
Hi Team,

I am getting this error on building the angular application with prod params. Please help.

ERROR in node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(29,35): error TS1005: ',' expected.
node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(29,41): error TS1005: ',' expected.
node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(42,59): error TS1005: ',' expected.
node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(42,65): error TS1005: ',' expected.
node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(271,33): error TS1005: ',' expected.
node_modules/@types/glob/node_modules/@types/minimatch/index.d.ts(271,39): error TS1005: ',' expected.

Findings:
**This was in version 3, which was working fin**e
function match(list: ReadonlyArray<string>, pattern: string, options?: IOptions): string[];

**This came in version 5 which created problem**
 function match(list: readonly string[], pattern: string, options?: IOptions): string[];

Angular version: 5
TS version: 2.7.2

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment, marked as answer]
@nitin1104 You have a very old version of the TS compiler. You either need to update your TS compiler to at least 4.1 or you have to use an older version of the definitions. To install the older version of the definitons, you can run:
```sh
npm i -D @types/minimatch@^3.0.5
```

**Reply by nitin1104** [reply]
Thanks
